diagrams/Design_Level_Class_Diagrams/ViewTourPackages.puml

```diff
@@ -104,6 +104,14 @@ package "Data Access Layer" <<Rectangle>> #LightCyan {
         + totalAmount: Decimal
     }
 
+    class TourEvent {
+            + tourEventId: UUID
+            + seq: String
+            + seqDesc: String
+            + eventId: String
+            + tourId: String
+        }
+
```

Comment:
The tour event entity is added so that tours can have multiple events, and events can belong to multiple tours, making events reusable at any time by all tours, and putting that event anywhere within the itinerary.